heim-process/src/sys/linux/process/procfs/stat.rs

```diff
@@ -61,7 +61,10 @@ impl FromStr for Stat {
         // `+ 2` is for the ") " at the start
         let mut parts = leftover[comm_end + 2..].split_whitespace();
         let state = parts.try_next().and_then(|str| {
-            let chr = str.chars().next().ok_or_else(|| io::Error::from(io::ErrorKind::InvalidData))?;
+            let chr = str
+                .chars()
+                .next()
+                .ok_or_else(|| io::Error::from(io::ErrorKind::InvalidData))?;
```

Comment:
I'm not sure what happened here? `rustfmt` rules had changed since the last time?
Weirdly, CI says nothing about that change.

Reply:
That would be my guess - I didn't manually alter it but I do run rustfmt on save.

Reply:
I guess it's just `rustfmt` being weird then, thanks for clarifying!